### ISSUE TITLE
the program blocks access to the server vk.com

### ISSUE THREAD

**Issue by saber716rus** [celzero/rethink-app#1779]
Hello! I have a problem: the program blocks applications from accessing the VK server.
![Screenshot_2024-11-11-16-23-03-542_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/78c45744-16c5-4bfb-8367-74fd57e66aed)

![Screenshot_2024-11-11-16-23-11-353_com.vkontakte.android.jpg](https://github.com/user-attachments/assets/1a5e8b6f-2e46-4a99-b23b-5a90d7a6f8ab)

![Screenshot_2024-11-11-16-24-07-272_com.yandex.searchapp.jpg](https://github.com/user-attachments/assets/70e764cb-1a76-41ef-add2-cec9448f9230)

Версия rethink dns 0.5.5n

**Comment by saber716rus** [comment]
yes, I tried reinstalling the vk application. but it doesn't work in the browser either. I disable rethink dns, then access to VK is restored. the problem seems to be in the DNS of the rethink dns application itself.

**Comment by ignoramous** [comment]
I doubt it is DNS.

1. Can you check DNS Logs for `vk.com`?
2. Search for `vk.com` in Network logs. 

Tap on the entries that show up. A bottomsheet should come up with more information. The *footer* of the bottomsheet would show errors if any.

**Comment by saber716rus** [comment]
blocked api.vk.com![Screenshot_2024-11-13-09-16-45-813_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/0500c780-4a06-40fc-ab56-361bdb086e6a)

![Screenshot_2024-11-13-09-16-22-120_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/7b70b6d0-fe5a-45e4-8e0f-690b61755e7f)

![Screenshot_2024-11-13-09-16-10-996_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/e93ceead-345d-4e9e-af52-2b213458d3a8)

**Comment by saber716rus** [comment]
I gave permission, but the blocking continues.![Screenshot_2024-11-13-10-43-44-575_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/44d74b8e-6521-4939-878e-b12e47e956e6)

![Screenshot_2024-11-13-10-43-47-833_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/a17034ab-4ec8-40b0-995e-22813135f25e)

**Comment by saber716rus** [comment]
It's working!
![Screenshot_2024-11-13-17-24-03-803_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/d72a2435-bae1-4e43-9a7e-0eef6fe12e70)

![Screenshot_2024-11-13-17-23-42-729_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/ca3bc1b0-1f92-4d00-8315-9ccc861f9a19)

![Screenshot_2024-11-13-17-22-32-858_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/8236e7dc-282b-4029-a3ed-62ba87243c2f)

**Comment by saber716rus** [comment]
Everything seems to be working, but one thing is still blocked.
![Screenshot_2024-11-13-17-24-03-803_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/c6c368c2-2dab-4aaa-978b-1fe240eaf498)

![Screenshot_2024-11-13-17-23-42-729_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/df593aeb-894b-4e6b-ab10-59ca741da83d)

![Screenshot_2024-11-13-17-30-04-343_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/05c398d8-bc97-41a4-9f61-4928b70a127a)

![Screenshot_2024-11-13-17-22-32-858_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/0676307a-b4e6-4efb-97b3-3c289124ca9a)

**Comment by ignoramous** [comment]
The "yellow" colour means, the domain was *allowed*. It may however be blocked by IP rule / per-app rules. So, you'll need to check *Network* logs for `vk`.

Alternatively, *allow* / *trust* `vk.com` and `*.vk.com` in *Configure* -> *Firewall* -> *IP & Port rules* -> swipe to *Domain rules* -> tap on `+` button at the footer.

**Comment by saber716rus** [comment]
I'm sorry, but you didn't quite understand. I realized that api.vk.com now it is allowed but here login.vk.com it is not allowed and the screenshot confirms it.![Screenshot_2024-11-13-17-30-04-343_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/f79e1a3c-eed6-4595-8cfe-9be121daa401)

**Comment by ignoramous** [comment]
I understood. Can you please add `*.vk.com` (note the wildcard `*`) to allowlist and see if it unblocks `login.vk.com`?

**Comment by saber716rus** [comment]
I did, it didn't help.![Screenshot_2024-11-14-08-18-29-544_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/8842d2e8-5c85-466e-90ae-4efcfb95b059)

**Comment by ignoramous** [comment]
Hm. Strange. "send fail" usually means either there's no internet or the DNS servers (here, `max.rethinkdns.com`) is not reachable. Not that the DNS server is refusing to resolve the domain (here, `login.vk.com`).

I'll test the (resolver) server code but it is unlikely that it is at fault.

**Comment by saber716rus** [comment]
That's right, what's strange is that on login.vk.com a mistake. By the way, on vk.com there is a lockdown again, and api.vk.com allowed.![Screenshot_2024-11-14-19-25-52-154_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/25e2c2f3-9431-4376-8383-a7067242720d)

![Screenshot_2024-11-14-19-25-39-031_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/c571fedc-1c8f-4361-bef0-3171f1bad414)

![Screenshot_2024-11-14-19-26-12-442_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/a1079a05-a446-4fef-a1f5-c0c4fb767211)

**Comment by ignoramous** [comment]
Hm either it is a bug in the version of Rethink you're using or the rules aren't setup appropriately. Can you show screenshot of global domain name rules UI which is at *Configure* -> *Firewall* -> *IP & Port rules* -> swipe to "Domain rules"?

If it is a bug with Rethink, then wait for `v055o` to launch (which should happen in a week or so).

**Comment by saber716rus** [comment]
here is another domain related error vk.com![Uploading Screenshot_2024-11-15-09-14-33-823_com.celzero.bravedns.jpg …]()

**Comment by saber716rus** [comment]
here is another domain related error vk.com![Screenshot_2024-11-15-09-14-33-823_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/53080d55-58fb-4add-b673-354638193990)

**Comment by ignoramous** [comment]
> another domain related error vk.com

That isn't a failure. It is a "no answer" for a HTTPS/SCVB query, probably sent by Chrome.

> interesting information has been dropped in the telegram channel of your application

Sorry, what information? I didn't quite catch what's written in Russian (with Google Translate).

**Comment by saber716rus** [comment]
https://t.me/rethinkdns/126222
Link to the Telegram post from the domain vk.com , where the response of one of the users is copied, which explains in detail about the domain vk.com

**Comment by saber716rus** [comment]
domain login.vk.com blocked.![Screenshot_2024-11-17-08-51-08-072_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/7cb0ed2d-e967-4048-a6a8-76893e75d378)

![Screenshot_2024-11-17-08-51-05-081_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/f2e28592-2d48-4866-99a9-a7286762847c)

**Comment by saber716rus** [comment]
<details><summary>here is another http service binding error</summary>

![Screenshot_2024-11-17-15-21-30-106_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/cbd637bc-ff3f-42b6-b267-9b672be383ee)

![Screenshot_2024-11-17-15-21-34-502_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/1008db99-379a-45e5-b755-7f92f97806c5)

![Screenshot_2024-11-17-15-21-42-572_com.celzero.bravedns.jpg](https://github.com/user-attachments/assets/5eec35ba-449f-4377-a52f-8b6ce4f23f60)

</details>

**Comment by ignoramous** [comment]
@saber716rus These "no answer" for HTTPS/SVCB records isn't what's blocking `vk.com`.

~~`v055o`~~ `v055p` is out: https://github.com/celzero/rethink-app/releases/tag/v0.5.5p Will you please check if `vk.com` works now? We have improved a LOT of stuff around connection handling, so I'm hopeful that whatever you were seeing has fixed itself.

**Comment by saber716rus** [comment]
> @saber716rus These "no answer" for HTTPS/SVCB records isn't what's blocking `vk.com`.
> 
> ~~`v055o`~~ `v055p` is out: https://github.com/celzero/rethink-app/releases/tag/v0.5.5p Will you please check if `vk.com` works now? We have improved a LOT of stuff around connection handling, so I'm hopeful that whatever you were seeing has fixed itself.

hello. I checked it out. The vk app and website are working. thanks for solving the problem. you can close it.

**Comment by saber716rus** [comment]
> @saber716rus These "no answer" for HTTPS/SVCB records isn't what's blocking `vk.com`.
> 
> ~~`v055o`~~ `v055p` is out: https://github.com/celzero/rethink-app/releases/tag/v0.5.5p Will you please check if `vk.com` works now? We have improved a LOT of stuff around connection handling, so I'm hopeful that whatever you were seeing has fixed itself.

I checked it out. blockages vk.com Not anymore. There is also no error "These "no answer" for HTTPS/SVCB "

**Comment by ignoramous** [comment]
Thanks for confirming that this issue has resolved in `v055p` and later versions. Closing...

btw, v055r is latest: https://github.com/celzero/rethink-app/releases/tag/v0.5.5r
